packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -48,8 +54,8 @@ export class ChoiceGroupOptionBase extends React.Component<IChoiceGroupOptionPro
       hasImage: !!imageSrc,
       checked,
       disabled,
-      imageIsLarge: !!imageSrc && (imageSize.width > 71 || imageSize.height > 71),
-      imageSize,
+      imageIsLarge: !!imageSrc && (imageSize!.width > LARGE_IMAGE_SIZE || imageSize!.height > LARGE_IMAGE_SIZE),
+      imageSize, // avoid unnecessary mutation for better style caching
```

Comment:
Might be better to have this comment up by defaultProps

Reply:
i like it here better bc if people see perf issue with component/style, they will come to this method, and then this comment

Reply:
But it doesn't explain specifically why the default value needs to be in defaultProps instead of provided inline.

Reply:
it doesn't have to be using `defaultProps`, as long as we don't mutate it on every render (which is what we do before).

Reply:
Sorry to keep being nitpicky about this, but the comment as it is now doesn't explain *how* we're ensuring unnecessary mutation is avoided (or under what circumstances it was happening before).

Reply:
To take that one step further, is there an explicit description of behavior that `defaultProps` will never mutate? It may mutate less, but unless there's an explicit React specification for behavior, I wonder if it wouldn't be better to make this completely explicit, defined a module const, and assign it to `defaultProps`.

Reply:
That's also why I wanted something like that included with defaultProps, so someone doesn't change how the default is provided (without happening to see this comment later) and reintroduce the issue.

Reply:
(I'm not sure how much using `static` in a class really guarantees only one instance is ever created in JS. Maybe this isn't an issue.)

Reply:
@ecraig12345 - ah ok i get your concern now. will update the comment
@JasonGore - when it's defined as static, there is only one instance.